cpp/log/cert_test.cc

```diff
@@ -380,11 +380,14 @@ TEST_F(CertTest, Extensions) {
 
   EXPECT_EQ(Cert::TRUE, pre.HasCriticalExtension(cert_trans::NID_ctPoison));
 
-  EXPECT_EQ(Cert::FALSE, leaf.HasBasicConstraintCATrue());
-  EXPECT_EQ(Cert::TRUE, ca.HasBasicConstraintCATrue());
-
-  EXPECT_EQ(Cert::TRUE, ca_pre.HasExtendedKeyUsage(
-                            cert_trans::NID_ctPrecertificateSigning));
+  StatusOr<bool> leaf_ca_status = leaf.HasBasicConstraintCATrue();
+  EXPECT_TRUE(leaf_ca_status.ok() && leaf_ca_status.ValueOrDie() == false);
```

Comment:
Same as for my `cpp/log/ct_extensions_test.cc` comment, could let the `OrDie` work its magic in tests, and just `EXPECT_FALSE(leaf_ca_status.ValueOrDie());`? And if it's referenced only once, could use it as it used to be:

```C++
EXPECT_FALSE(leaf.HasBasicConstraintCATrue().ValueOrDie);
```

Similar thing for the checks below...